### PR TITLE
Feat: Page Breadcrumbs

### PR DESCRIPTION
**Pull request by JeangelLF**
## Ticket

https://github.com/corona-school/project-user/issues/1322

## What was done?

- Added Breadcrumb component
- Added hook with mapped breadcrumb items
- Included Breadcrumb component in Appointments, Matches, Courses pages / Also removed the back button from those pages

Bildschirmfoto 2024-10-22 um 11 49 11

### CODE REVIEW

File: src/components/Breadcrumb.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                    <>
+                        <BreadcrumbPath>
+                            <BreadcrumbLink to="/start">{t('navigation.label.start')}</BreadcrumbLink>
+                        </BreadcrumbPath>
+                        <BreadcrumbSeparator>
+                            <IconSlash />
+                        </BreadcrumbSeparator>
+                    </>
+                )}
+                {(breadcrumb || items).map((item, index) => {
```
Comment (realmayus) [inline src/components/Breadcrumb.tsx:93]: `|| items` is redundant as far as I can tell

File: src/lang/de.json [hunk 2]
```diff
@@ -2095,7 +2095,16 @@
...
             "forStudents": "Wissens-Center",
-            "forPupils": "Wissens-Center"
+            "forPupils": "Wissens-Center",
+            "requestMatch": "Ein neues Lernpaar bilden",
+            "createCourse": "Kurse anlegen",
+            "createAppointment": "Termin erstellen",
+            "editAppointment": "Termin bearbeiten",
+            "settings": "Einstellungen",
+            "notifications": "Benachrichtigungen",
+            "systemNotifications": "System Benachrichtigungen",
```
Comment (realmayus) [inline src/lang/de.json:2105]: System-Benachrichtigungen

File: src/lang/de.json [hunk 3]
```diff
@@ -2095,7 +2095,16 @@
...
+            "newsletterNotifications": "Newsletter Benachrichtigungen",
```
Comment (realmayus) [inline src/lang/de.json:2106]: Newsletter-Benachrichtigungen

File: src/pages/Appointment.tsx [hunk 4]
```diff
@@ -120,8 +125,11 @@ const Appointment: React.FC<AppointmentParams> = ({ startMeeting }) => {
         return '/appointments';
     };
 
+    const appointmentTile = data?.appointment?.title || t('appointment.appointmentTile.lecture', { position: data?.appointment?.position });
+    console.log({ appointmentTile });
```
Comment (realmayus) [inline src/pages/Appointment.tsx:129]: probably should leave this out

**Review by realmayus — APPROVED** [review]
Nice work!

File: src/components/Breadcrumb.tsx (hunk 1 above)
Comment (JeangelLF) [reply src/components/Breadcrumb.tsx]: Yep, it's. Thank you!

**Pull request merged**